src/core/Akka.Remote.Tests.Performance/Transports/TestTransportRemoteMessagingThroughputSpec.cs

```diff
@@ -31,8 +34,16 @@ public override Config CreateActorSystemConfig(string actorSystemName, string ip
             var remoteAddress = $"test://{actorSystemName}@{ipOrHostname}:{port}";
             var bindingConfig =
                 ConfigurationFactory.ParseString(@"akka.remote.test.local-address = """+ remoteAddress +@"""");
+            var registryKeyConfig = ConfigurationFactory.ParseString($"akka.remote.test.registry-key = {_registryKey}");
+
+            return registryKeyConfig.WithFallback(bindingConfig.WithFallback(baseConfig));
+        }
 
-            return bindingConfig.WithFallback(baseConfig);
+        public override void Cleanup()
+        {
+            // force all content logged by the TestTransport to be released
+            AssociationRegistry.Get(_registryKey).Reset();
```

Comment:
Resolves the memory leak from#1556 - this code allows us to retrieve the `AssociationRegistry` for this spec and the `Reset` method dumps all of the retained logs (which add up very quickly - the byte payload for literally every write gets logged) and `AssociationHandle` instances.